week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>Are you focusing on specific metrics for the course? They could be a bit overwhelmed by all these terms.</p><p> </p><p>I would definitely leave them in for those how are interested to learn more but I would probably focus only on a subset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>i will focus on accuracy, and confusion matrices, then I will probably go over the rest briefly. We will see how the time goes haha.</p>